### ISSUE TITLE
I can do that.

### ISSUE THREAD

**Issue by palemieux** [w3c/PNG-spec#135]
I can do that.
Would you be okay if I do that in a separate pull request, though?

_Originally posted by @ProgramMax in https://github.com/w3c/PNG-spec/pull/128#discussion_r905389847_

**Comment by ProgramMax** [comment]
To add context here, the class named "Definition" was from a style sheet that was added to one edition of the spec and removed in a subsequent revision to that edition. There is no longer a matching class named "Definition".

**Comment by ProgramMax** [comment]
Also `<span class="chunk">`

**Comment by palemieux** [comment]
> Also <span class="chunk">

I think this is a separate issue since we might actually want to style reference chunks differently.

I am not too excited by `<code><a>cICP</a></code>`... maybe there is something better that both (a) conveys visually that it is a chunk name and (b) links to the chunk definition?

**Comment by ProgramMax** [comment]
I agree with you. Even if we don't plan to use it now, it provides an easy access point into styling chunks.

I think there are several others of these to consider, still. For example, `<table class="Regular">` and `<td class="Regular">`?

**Comment by ProgramMax** [comment]
There is also `<figcaption class="Figuretitle">`, `<span class="figref">`, and `<span class="tableref">`...

**Comment by ProgramMax** [comment]
And `<br class="xhtml" />` :D

**Comment by svgeesus** [comment]
> Also `<span class="chunk">`

I think we should keep that one tbh

**Comment by palemieux** [comment]
> > Also `<span class="chunk">`
> 
> I think we should keep that one tbh

We could do `<a class="chunk">cICP</a>`?

**Comment by ProgramMax** [comment]
Note that in the spec there is an example comparing CHNK and cHNk as theoretical, example chunks. These are marked as '<span class="chunk">' and cannot be marked as '<a class="chunk">'. I left these as-is.
